metadata-ws/src/test/resources/features/file.feature

```diff
@@ -34,3 +34,74 @@ Feature: file object
     """
     When I request GET /files
     Then the response should contain one file
+
+  Scenario: attempt to get a file of an analysis of an unreleased study must fail
```

Comment:
The tests have now been significantly updated and expanded. They now include all combinations of null/past/future release dates for child entities of a Study.

The Study itself does not need to be tested because there are already existing tests in `study.feature` which check everything about Study's `releaseDate`